api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessor.java

```diff
@@ -85,9 +93,11 @@ private void handleDecision(Claim claim, PaymentCycle previousPaymentCycle, Paym
             createMakePaymentMessage(currentPaymentCycle);
         } else if (claim.getClaimStatus() == ACTIVE) {
             if (shouldExpireClaim(decision, previousPaymentCycle, currentPaymentCycle)) {
-                expireClaim(claim);
+                updateClaimStatus(claim, ClaimStatus.EXPIRED);
+                eventAuditor.auditExpiredClaim(claim);
+                claimMessageSender.sendReportClaimMessage(claim, emptyList(), UPDATED_FROM_ACTIVE_TO_EXPIRED);
```

Comment:
shouldn't we pass the list of children's dob here? I know that we shouldn't have any children at this point, but just in case that changes...

Reply:
Updated to include the children from the decision.